test/unit/dummies/gpio/Gpio.c

```diff
@@ -2,7 +2,7 @@
 // Created by David P. Federl
```

Comment:
I suggest we remove comments like these completely. Git can tell us who changed a line. Also, no need to remove it now. Could be its own issue... run a regex over the whole project removing every line that says `// Created by .*$`

Reply:
I agree to you. If there are still some, they should be removed and are only there by accident!